utils/database.js

```diff
@@ -67,30 +67,26 @@ class Database {
   ) {
     let baniLength;
     switch (length) {
-      case "EXTRA_LONG":
-        baniLength = "existsBuddhaDal";
+      case 'EXTRA_LONG':
+        baniLength = 'existsBuddhaDal';
         break;
-      case "LONG":
-        baniLength = "existsTaksal";
+      case 'LONG':
+        baniLength = 'existsTaksal';
         break;
-      case "MEDIUM":
-        baniLength = "existsMedium";
+      case 'MEDIUM':
+        baniLength = 'existsMedium';
         break;
-      case "SHORT":
-        baniLength = "existsSGPC";
+      case 'SHORT':
+        baniLength = 'existsSGPC';
```

Comment:
These variables like `EXTRA_LONG`, `LONG`, etc are being used multiple times in the application as a plain string. It is strongly recommended to put them in a file as constant variables and use it instead.

Reply:
Fixed